src/o365/spfx/commands/project/project-externalize/rules/DynamicRule.ts

```diff
@@ -101,41 +86,37 @@ export class DynamicRule extends BasicDependencyRule {
     });
   }
 
-  private getFileUrl(packageName: string, version: string, filePath: string) {
-    return `https://unpkg.com/${packageName}@${version}/${filePath}`;
-  }
-
-  private testUrl(url: string): Promise<boolean> {
-    return new Promise<boolean>((resolve: (urlExists: boolean) => void, reject: (err: any) => void): void => {
+  private testUrl(url: string): Promise<void> {
+    return new Promise<void>((resolve: () => void, reject: () => void): void => {
       request
         .head({ url: url, headers: { 'x-anonymous': 'true' } })
         .then(() => {
-          resolve(true);
+          return resolve();
         }, () => {
-          resolve(false);
+          return reject();
         });
     });
   }
 
   private getFilePath(packageName: string): string | undefined {
+    let result: string | undefined = undefined;
+
     const packageJsonFilePath: string = `node_modules/${packageName}/package.json`;
 
     try {
       const packageJson: { module?: any, main?: any } = JSON.parse(fs.readFileSync(packageJsonFilePath, 'utf8'));
 
       if (packageJson.module) {
-        return packageJson.module;
+        result = packageJson.module;
       }
       else if (packageJson.main) {
-        return packageJson.main;
-      }
-      else {
-        return undefined;
+        result = packageJson.main;
       }
+    } catch {
+      // file doesn't exist, giving up
     }
-    catch { // file doesn't exist, giving up
-      return undefined;
-    }
+    
+    return result;
```

Comment:
Had to introduce the return at the very end due to c8 issue. See: https://github.com/bcoe/c8/issues/62